kyverno/allow-capabilities.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: allow-add-capabilities
+  namespace: openebs
+  annotations:
+    policies.kyverno.io/category: Pod Security Standards
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+          Provides a list of capabilities that are allowed to be added to a container.
+spec:
+  validationFailureAction: audit
+  background: true
+  rules:
+    - name: capabilities
+      match:
+        resources:
+          name: "openebs-*"
+          kinds:
+            - Pod
+      validate:
+        message: >-
+          Default set of capabilities are allowed.        
+        pattern:
+          spec:
+            containers:
+              - =(securityContext):
+                  =(capabilities):
+                    =(add): "*"
+            =(initContainers):
```

Comment:
Do we have any initContainers created from openebs? If not, do we need this pattern here?

Reply:
We can decide its a standard policy pattern which includeds the init containers as well in [kyverno docs.](https://kyverno.io/policies/pod-security/)